autoload/lsc/complete.vim

```diff
@@ -272,6 +272,7 @@ function! s:CompletionItem(completion_item) abort
           \ 'snippet': item.word,
           \ 'snippet_trigger': item.word
           \ })
+    let l:item.word = a:completion_item.label
```

Comment:
If `completion_item`'s `insertTextFormat` is snippet, we should show `label` instead of `insertText` for pum maybe.